app/views/user_bets/new.html.erb

```diff
@@ -0,0 +1,29 @@
+<% if current_user && @option %>
```

Comment:
Checking for a logged in user should happen at the controller level.
Could we also make this work if the option doesnt exist